### PR TITLE
examples: DaeDaLus nitf parser demo

### PR DESCRIPTION
**Pull request by kaoudis**

### CODE REVIEW

File: polytracker/src/passes/bb_splitting_pass.cpp
Comment (josephsweeney) [inline polytracker/src/passes/bb_splitting_pass.cpp:1]: I think we can revert this, there was a one line change that I originally needed for M1 compatibility and I think this file is just no longer relevant. I can make the change and test if you want.

Comment (kaoudis) [reply polytracker/src/passes/bb_splitting_pass.cpp:1]: sure, go for it! thank you

Comment (josephsweeney) [reply polytracker/src/passes/bb_splitting_pass.cpp]: Done. I couldn't find the PR that remove that file though, I thought it must have been the sections PR but I don't see it. 🤔 Probably best to wait for CI before a merge.

**Review by josephsweeney — APPROVED** [review]

**Review by ESultanik — APPROVED** [review]

**Pull request merged**
